api/TOC.md

```diff
@@ -17802,6 +17803,7 @@
 ###### [HasTitleMaster](PowerPoint.Presentation.HasTitleMaster.md)
 ###### [HasVBProject](PowerPoint.Presentation.HasVBProject.md)
 ###### [InMergeMode](PowerPoint.Presentation.InMergeMode.md)
+###### [InMergeMode](PowerPoint.Presentation.IsFullyDownloaded.md)
```

Comment:
@lindalu-MSFT  I think this should be 
`###### [IsFullyDownloaded](PowerPoint.Presentation.IsFullyDownloaded.md)`
Instead of `###### [InMergeMode](PowerPoint.Presentation.IsFullyDownloaded.md)`

Reply:
HI, I fixed this in https://github.com/MicrosoftDocs/VBA-Docs/pull/1630 and merged. It should work now, if not clear your cache and try again.